client-app/src/Components/StatusDisplayPage.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react'; // Do we need this file anymore?
```

Comment:
It's not needed anymore. You can delete all files with this name with .js and  .css extensions